demo/runDemo.json5

```diff
@@ -0,0 +1,4 @@
+{ // Run configuration file for demonstration pipeline - can be run directly from command line via runFluidPipeline %forgiving-data/runDemo.json5
+    loadDirectory: "%forgiving-data/demo/pipelines",
+    execPipeline: ["fluid.pipelines.WeCount-ODC-fileOutput", "fluid.pipelines.WeCount-ODC-synthetic"]
```

Comment:
Every time I look at this line after a while, my first instinct is to execute pipelines listed in the array in sequence rather than executing a merged pipeline. I'm thinking if renaming the key to `execMergedPipeline` would reduce the confusion. What do you think?

Reply:
Good idea

Reply:
Thanks for adding the extra config `execMergedPipeline`. Shall it be used at this line as multiple pipelines are presented for a merge before the execution?

Reply:
Thanks, now updated